app/views/contacts/show.html.slim

```diff
@@ -0,0 +1,50 @@
+.row
+  .col-xs-12
+    h1= t_title(:show)
+
+.row
+  .col-xs-12
+    dl.dl-horizontal
+      - if kind == 'person'
+        dt= t_attr(:first_name)
+        dd= @contact.first_name
+
+        dt= t_attr(:last_name)
+        dd= @contact.last_name
+
+        dt= t_attr(:title)
+        dd= @contact.title
+
+        dt= t_attr(:gender)
+        dd= @contact.gender
+
+        dt= t_attr(:birthday)
+        dd= @contact.birthday
+
+        dt= t_attr(:birthplace)
+        dd= @contact.birthplace
+
+      dt= t_attr(:organisation)
+      dd= @contact.organisation
+
+      dt= t_attr(:org_role)
+      dd= @contact.org_role
+
+- if kind == 'person'
```

Comment:
@casaper I am not scaffolding another contact model. Volunteers have for the time being address fields, email and phone number, but that is the same as it is now with clients.

But, I think @kitsane is talking about the expertise, hobby and interest fields. I have similar fields, but those aren't contact fields, are they?